news/2018-03-27-the-qat-gazette-march-2018.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "The QAT Gazette: March 2018"
+date: 2018-03-27 09:00:00 +0000
+---
+Restructuring of the Quality Assurance Team, more communication and transparency regarding decision-making, and a greater focus on actually ensuring quality in the Qualified section again... Big changes to discuss this month, so read inside to find out all the details!
+
+![](/wiki/shared/news/banners/qat-blog-logo.png)
+
+## Opening 
+by [JBHyperion](https://osu.ppy.sh/users/4879508)
+
+Welcome back everyone! JBHyperion here with another month's worth of news and developments!
+
+As some of you may already be aware, it's been another busy month in our fair community, though you've hopefully come to expect that - these articles wouldn't be very exciting if there was nothing interesting to talk about after all! Thankfully, it's also been a month of tremendous growth. With growth comes change, and with changes comes an exciting opportunity for us to tell you all about it! 
```

Comment:
wouldn't be so exciting
With growth comes changes, and with changes come an exciting opportunity for us to tell you all about it!

Reply:
Either are acceptable here